lingua_franca/res/countries.json

```diff
@@ -0,0 +1,3227 @@
+        "official_name_es": "Andorra",
+        "official_name_cn": "安道尔",
+        "official_name_en": "Andorra",
+        "official_name_ru": "Андорра",
+        "Language": "ca"
+    },
+    {
+        "ISO3166-1-Alpha-3": "AGO",
+        "ISO3166-1-Alpha-2": "AO",
+        "official_name_fr": "Angola",
+        "ISO4217-currency_name": "Kwanza",
+        "ISO4217-currency_alphabetic_code": "AOA",
+        "official_name_ar": "أنغولا",
+        "official_name_es": "Angola",
+        "official_name_cn": "安哥拉",
+        "official_name_en": "Angola",
+        "official_name_ru": "Ангола",
+        "Language": "pt-AO"
+    },
+    {
+        "ISO3166-1-Alpha-3": "AIA",
+        "ISO3166-1-Alpha-2": "AI",
+        "official_name_fr": "Anguilla",
+        "ISO4217-currency_name": "East Caribbean Dollar",
+        "ISO4217-currency_alphabetic_code": "XCD",
+        "official_name_ar": "أنغيلا",
+        "official_name_es": "Anguila",
+        "official_name_cn": "安圭拉",
+        "official_name_en": "Anguilla",
+        "official_name_ru": "Ангилья",
+        "Language": "en-AI"
+    },
+    {
+        "ISO3166-1-Alpha-3": "ATA",
+        "ISO3166-1-Alpha-2": "AQ",
+        "official_name_fr": "Antarctique",
+        "ISO4217-currency_name": "No universal currency",
+        "ISO4217-currency_alphabetic_code": "",
+        "official_name_ar": "أنتاركتيكا",
+        "official_name_es": "Antártida",
+        "official_name_cn": "南极洲",
+        "official_name_en": "Antarctica",
+        "official_name_ru": "Антарктике"
+    },
+    {
+        "ISO3166-1-Alpha-3": "ATG",
+        "ISO3166-1-Alpha-2": "AG",
+        "official_name_fr": "Antigua-et-Barbuda",
+        "ISO4217-currency_name": "East Caribbean Dollar",
+        "ISO4217-currency_alphabetic_code": "XCD",
+        "official_name_ar": "أنتيغوا وبربودا",
+        "official_name_es": "Antigua y Barbuda",
+        "official_name_cn": "安提瓜和巴布达",
+        "official_name_en": "Antigua and Barbuda",
+        "official_name_ru": "Антигуа и Барбуда",
+        "Language": "en-AG"
+    },
+    {
+        "ISO3166-1-Alpha-3": "ARG",
+        "ISO3166-1-Alpha-2": "AR",
+        "official_name_fr": "Argentine",
+        "ISO4217-currency_name": "Argentine Peso",
+        "ISO4217-currency_alphabetic_code": "ARS",
+        "official_name_ar": "الأرجنتين",
+        "official_name_es": "Argentina",
+        "official_name_cn": "阿根廷",
+        "official_name_en": "Argentina",
+        "official_name_ru": "Аргентина",
+        "Language": "es-AR"
+    },
+    {
+        "ISO3166-1-Alpha-3": "ARM",
+        "ISO3166-1-Alpha-2": "AM",
+        "official_name_fr": "Arménie",
+        "ISO4217-currency_name": "Armenian Dram",
+        "ISO4217-currency_alphabetic_code": "AMD",
+        "official_name_ar": "أرمينيا",
+        "official_name_es": "Armenia",
+        "official_name_cn": "亚美尼亚",
+        "official_name_en": "Armenia",
+        "official_name_ru": "Армения",
+        "Language": "hy"
```

Comment:
`hy-AM`?

Reply:
Not sure if this is accurate and it would be impossible to validate. We can stick with "official" dataset unless/until someone who knows corrects it.